opendevin/sandbox/sandbox.py

```diff
@@ -178,7 +179,12 @@ def close(self):
         self.closed = True
 
     def stop_docker_container(self):
-        docker_client = docker.from_env()
+
+        try:
+            docker_client = docker.from_env()
+        except docker.errors.DockerException as e:
+            raise e
```

Comment:
should we print something here too?

Reply:
Implemented the change. The trace is Session > AgentController > DockerInteractive > restart_docker_container > stop_docker_container. Initially considered session.py, but it's more effective to advise on docker validation directly in stop_docker_container().